leakcanary-android-core/src/main/java/leakcanary/internal/Notifications.kt

```diff
@@ -49,7 +49,12 @@ internal object Notifications {
     if (!canShowNotification) {
       return
     }
-    val builder = Notification.Builder(context)
+
+    val builder = if (SDK_INT >= O) {
+      Notification.Builder(context, type.name)
```

Comment:
@SylvainGirod the second parameter here is a channel id, not a notification group.

https://developer.android.com/reference/kotlin/android/app/Notification.Builder?hl=en#%3Cinit%3E(android.content.Context,%20kotlin.String)

Did you mean to call setGroup instead?

Reply:
@pyricau This may be out of scope, but since API 26, the constructor for `Notification.Builder` requires a `String` for the channelId as the second parameter. I thought it could make sense to make the change necessary to handle pre and post API 26 as it should be.
If you think this should be split, I can do it easily.

`setGroup` is called on line 90.

Reviewing again this PR made me look that on line 89 I forgot to delete the call to `setChannelId`.
Before doing this change, I will wait for your decision about this change.

Reply:
Makes sense, I didn't look carefully enough, thanks for the explanation.

>  I forgot to delete the call to setChannelId

We can't delete it without updating all calls to `Notifications.buildNotification()`, there are quite a few. Should maybe update this so that the builder is built in one place and you can provide a lambda to add to it, or something like that.

However that can come later (feel free to open a PR if you want), I'll merge his one.